src/api/route-services/gem2s.js

```diff
@@ -82,12 +81,7 @@ class Gem2sService {
   }
 
   static async gem2sCreate(experimentId, body, authJWT) {
-    const { shouldRun, gem2sHash: paramsHash } = body;
-
-    if (!shouldRun) {
-      logger.log('Gem2s create call ignored');
-      return OK();
-    }
+    const { paramsHash } = body;
 
```

Comment:
add logging GEM2S running

Reply:
Add loggging info to help debugging. If GEM2S creation failed, what info will help with debugging it?